I don't think so:

```
$ ./result/bin/geteduroam-gui
pkg-config, failed with: exec: "pkg-config": executable file not found in $PATH and stderr:
panic: Path for library: cairo not found. Please set the path to this library shared object file manually with env variable: PUREGOTK_CAIRO_PATH or PUREGOTK_LIB_FOLDER. Or make sure pkg-config is setup correctly

goroutine 1 [running]:
github.com/jwijenbergh/puregotk/internal/core.GetPath({0xbfae39, 0x5})
        github.com/jwijenbergh/puregotk@v0.0.0-20240527152221-b010cb9a4ca8/internal/core/core.go:169 +0x2ab
github.com/jwijenbergh/puregotk/v4/cairo.init.0()
        github.com/jwijenbergh/puregotk@v0.0.0-20240527152221-b010cb9a4ca8/v4/cairo/main.go:554 +0x1f
```

This is coming from this go library: https://pkg.go.dev/github.com/jwijenbergh/puregotk#readme-library-loading

--- a/pkgs/by-name/ge/geteduroam-cli/package.nix
+++ b/pkgs/by-name/ge/geteduroam-cli/package.nix
@@ -1,0 +1,45 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "geteduroam-cli";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+    owner = "geteduroam";
+    repo = "linux-app";
+    tag = version;
+    hash = "sha256-9+IjrHg536/6ulj94CBhYWY0S3aNA7Ne4JQynMmsLxE=";
+  };
+
+  vendorHash = "sha256-9SNjOC59wcEkxJqBXsgYClHKGH7OFWk3t/wMPLANAy0=";
+
+  subPackages = [
+    "cmd/geteduroam-cli"
+  ];
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${placeholder "out"}/bin/geteduroam-cli";
+  versionCheckProgramArg = [ "--version" ];
+  doInstallCheck = true;
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "CLI client to configure eduroam";
+    mainProgram = "geteduroam-cli";
+    homepage = "https://github.com/geteduroam/linux-app";
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ viperML ];
+    platforms = lib.platforms.linux;
+    changelog = "https://github.com/geteduroam/linux-app/releases/tag/${version}";
+  };
+}

